gapii/cc/spy.cpp

```diff
@@ -108,6 +108,9 @@ const uint32_t kStartMidExecutionCapture =  0xdeadbeef;
 const int32_t  kSuspendIndefinitely = -1;
 
 const uint8_t kCoreAPI = 0;
```

Comment:
Actually come to think of it, I might be able to emit something into vulkan_spy.h / gles_spy.h and handle it that way. I will give it a shot.